src/Symfony/Bridge/Twig/DataCollector/TwigDataCollector.php

```diff
@@ -16,6 +16,7 @@
 use Symfony\Component\HttpKernel\DataCollector\DataCollector;
 use Symfony\Component\HttpKernel\DataCollector\LateDataCollectorInterface;
 use Twig\Environment;
+use Twig\Error\ErrorLoader;
```

Comment:
it's LoaderError, isn't it?

Reply:
Yes it is https://github.com/twigphp/Twig/blob/2.x/src/Error/LoaderError.php